### PR TITLE
VxWorks 7 RTP support

### PR DESCRIPTION
**Pull request by johnsonjh**
* VxWorks 7 RTP support.
* Tested with VxWorks 7 version 21.03 (Clang 11) through 23.09 (Clang 16).

### CODE REVIEW

File: src/sirfilesystem.c [hunk 1]
```diff
@@ -321,7 +321,9 @@ char* _sir_getappfilename(void) {
             break;
         }
 # else
-#  error "no implementation for your platform; please contact the author."
+#  if !defined(__VXWORKS__) /* XXX(jhj): Implement for VxWorks */
```
Comment (aremmell) [inline src/sirfilesystem.c:324]: You should create an issue for this so you don't forget

File: src/sirfilesystem.c [hunk 2]
```diff
@@ -321,7 +321,9 @@ char* _sir_getappfilename(void) {
...
+#   error "no implementation for your platform; please contact the author."
```
Comment (aremmell) [inline src/sirfilesystem.c:325]: We should change all of these messages to read "the maintainers."

File: .gitlab-ci/vxworks7/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2018-current Jeffrey H. Johnson <trnsz@pobox.com>
```
Comment (aremmell) [inline .gitlab-ci/vxworks7/Dockerfile:2]: Didn't you recently add a PR that removed the 'current' syntax?

Comment (johnsonjh) [reply .gitlab-ci/vxworks7/Dockerfile:2]: Yes, I've not rebased this yet.  Still a WIP.

**Comment by johnsonjh** [comment]
I'll just close this until it's ready.

**Pull request closed without merging**